src/app.js

```diff
@@ -1 +1,18 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const [oldPath, newPath] = process.argv.slice(2);
+
+oldPath !== newPath
+  ? fs.copyFile(`${oldPath}`, `${newPath}`, callback)
```

Comment:
```suggestion
  ? fs.copyFile(oldPath, newPath, callback)
```
do you really need interpolation here?